src/commonTest/kotlin/dev/romainguy/kotlin/math/MatrixTest.kt

```diff
@@ -523,6 +523,12 @@ class MatrixTest {
             Float4(4f, 8f, 12f, 16f)
         )
 
+        internal fun assertFloatEquals(expected: Float, actual: Float, delta: Float = 0.0001f) {
```

Comment:
It's not required, you can use `assertEquals(expected, actual, absoluteTolerance)` from `kotlin.test`.

Reply:
Fix [f883ae1](https://github.com/romainguy/kotlin-math/pull/46/commits/f883ae15a21bb2ea4e2df20ceaf8272089c89bf6)